index.js

```diff
@@ -62,6 +61,10 @@ function getImportPath(importCache, pantry, ingredientName) {
  * looked up in the locations named by `config.pantrySearchPaths`. The first
  * matching pantry found is cached and the above process performed.
  *
+ * @param {*} NODE_SASS_NULL The object to return when node-sass should do its
+ *                           thing. You should pass require('node-sass').NULL
```

Comment:
Please wrap to a two character indent from the start of the `@param` (see line 70 in your patch for an example)